www/src/components/Toc.jsx

```diff
@@ -10,9 +10,9 @@ const Toc = ({ data }) => {
           <li key={heading.url}>
             <a href={heading.url}>{heading.title}</a>
             {!!heading.items && (
-              <ul className="pgn-doc__toc-list">
+              <div className="pgn-doc__toc-list">
```

Comment:
[clarification] Is this change to avoid having a `ul` as a direct child of another `ul`?

Reply:
Fixed it in the code rather than just renaming `ul` to `div`, turns out we don't need that wrapper at all